Doc/library/argparse.rst

```diff
@@ -2246,8 +2246,8 @@ Exiting methods
 
 .. method:: ArgumentParser.error(message)
 
-   This method prints a usage message including the *message* to the
-   standard error and terminates the program with a status code of 2.
+   This method prints a usage message, including the *message*, to stderr
```

Comment:
I updated this to be consistent with the addition above. I'm happy to also write "standard error" if that's the agreed-upon convention. It seemed like there was mixed usage across the docs.

Reply:
The documentation for `deprecated` also refers to "standard error". On other hand, the documentation for `print_usage()` and `print_help()` refer to ``` :data:`sys.stdout` ``` (although this is slightly different context, because they talk about the value for `file`).

The `FileType` documentation refers to ``` :data:`sys.stdin` ``` and ``` :data:`sys.stdout` ```, but this can be ignored, because it is a different context and `FileType` will be deprecated in any way.

I think that leaving "standard error" is okay, but using ``` :data:`sys.stderr` ``` is better. Please update also the `deprecated` documentation.